### ISSUE TITLE
default shield-on option lost since a recent update

### ISSUE THREAD

**Issue by fche** [brave/brave-browser#23214]
## Description 

In versions prior to an update ~2-3 weeks ago (unknown exactly where this broke), my former default-shields-up setting (orange lion) has been broken.  Now, new websites default to shields-down (grey lion), and I can't find a setting to turn it back on.   This makes my browsing less secure than it was and than it should be.

## Brave version (brave://version info)
brave-browser-1.39.111-1.x86_64  (fedora 35)

Brave	1.38.119 Chromium: 101.0.4951.67 (Official Build) (64-bit) ￼
Revision	8888ee7a24e2c36661ddb9536c35b7d4852a3a98-refs/branch-heads/4951@{#1230}
OS	Linux

**Comment by rebron** [comment]
@fche Shields is on by default for all sites and there's no global setting to turn it off on new sites. We do have shields disabled on internal pages like the new tab page and settings. Sounds like you're saying every site you visit has Brave Shields off by default though?

**Comment by fche** [comment]
Yup, that's right, it's off by default on any new site I visit.
I can flick it on, and it stays on for future visits.
I wonder if some brave configuration item got messed up after many years of use & upgrading?

**Comment by rebron** [comment]
We were thinking it was just a UI issue where the icon was greyed out but Shields is still on so an issue with front end icon. But sounds like truly having Shields turned off for all new sites you visit.

**Comment by fche** [comment]
Yup, it is truly off (javascript crap works, even though global defaults are to block), etc.

**Comment by fche** [comment]
A possibly related phenomenon:  my brave://settings has become very short indeed:

![foo](https://user-images.githubusercontent.com/588885/171930691-acf04ba2-b4e8-46f9-a9be-4e20c535612e.png)

**Comment by rebron** [comment]
@fche That one above is an issue with minimum window, we're working on addressing it. If you expand window size, should see the rest of settings.

**Comment by MadhaviSeelam** [comment]
I couldn't reproduce the above issue  for fresh install and upgrade profile.  Shields toggle `ON` and `OFF` behavior works as expected in Win 11x64 and Ubuntu.  

Tested on Win 11x64  as below: 

Installed 1.38.119 Chromium 101.0.4951.67 and then upgraded to 1.39.111 Chromium: 102.0.5005.61 and both the versions have V2 Shields

- Install 1.38.119 Chromium 101.0.4951.67
- Launch brave
- open few websites ([CNN.com](http://cnn.com/), [BBC.com](http://bbc.com/), [theguardian.com](http://theguardian.com/))
- Verified all three websites opened with default Shields `ON`
- Toggle `OFF` for one of the sites ([theguardian.com](http://theguardian.com/))
- Close and relaunch the browser
- `CNN `and `BBC ` sites `Shields` setting correctly shows 'ON"
- Guardian site `Shield` displayed 'OFF' 
- Upgrade to 1.39.111 Chromium: 102.0.5005.61
- All 3 tabs are preserved
- Shields icon is 'ON' for `CNN `and `BBC `sites
- Shields icon is 'OFF' for `Guardian`
- Open two more new sites [brave.com](http://brave.com/) or [nyimes.com](http://nyimes.com/) and the toggle is 'ON`

Couldn't reproduce the issue with fresh install of 1.39.111. 

Tested upgrading V1 Shields>>V2 Shields with same above steps
- Upgrade from 1.37.116 Chromium: 100.0.4896.127 to 1.39.111 Chromium: 102.0.5005.61
- Results were same as above

**Comment by fche** [comment]
Thanks!  I wouldn't be surprised if it were some sort of corruption in my local settings, but have no idea where to look.  ISTR shield default-on vs. default-off being a user selected option, but can't find that now.  Maybe some configuration migration broke it?

**Comment by ziggiesandvik** [comment]
I'm having the same issue - I tried clearing "Site and Shields Settings" but that did not help.

`

1.39.122 Chromium: 102.0.5005.115 (Official Build) (x86_64)
--
Revision | 174dbe6e33bc81994fceb71d751be201d0b4803d-refs/branch-heads/5005_109@{#3}
OS | macOS Version 12.3.1 (Build 21E258)

`

**Comment by alexandrpunk** [comment]
I have the same issue

**Comment by Werve** [comment]
I think this problem is liked to https://github.com/brave/brave-browser/issues/23113

**Comment by gpowers01** [comment]
I'm having the same problem. Shields are down unless I manually enable them on each site I visit - then the setting persists.  How does one fix this issue?  This issue seems to have surfaced in the past few weeks.  Currently on Release Notes V1.39.122 (June 10th, 2022), which is the latest update.  Do I need to somehow reset my local settings?

**Comment by leotm** [comment]
if you want to avoid re-installing, this did the trick for me
- https://github.com/brave/brave-browser/issues/23113#issuecomment-1161708535

**Comment by fche** [comment]
(Clearing shields/sites settings did not work for me.)

**Comment by fche** [comment]
Another odd observation: on my linux/fedora brave installation, since the default-shield breakage, it often happens that visiting a new site HANGS with the lion-greyed, just waits while loading.  If during this time, I turn the shields on via the lion icon, the site load completes right away.

FWIW, on this browser, I also run a copy of ublock-origin.

**Comment by r-erd** [comment]
I had the same issue on MacOS Monterey 12.4 and my Android phone using Brave (synced).
I managed to fix it by leaving the sync chain and deleting the site and shield settings.
I also often had the issue of websites loading very slowly / hanging with the grey lion, which doesnt seem to happen anymore.

I also run ublock-origin in the same browser.

Tested it with random websites from google search results.

**Comment by eduncan911** [comment]
Version 1.39.122 Chromium: 102.0.5005.115 (Official Build) (64-bit)

- Exact same issue for days as OP.  I thought it was an extension and removed all - nope.
- There was nothing to "clear" in Shields, as suggested above.
- Ubuntu 21.10

**Comment by kjozwiak** [comment]
The above will require `1.41.95` or higher for `1.41.x` verification.

**Comment by eduncan911** [comment]
It would be nice to know what exactly "broke" with the last update of Brave.  I have never, not once, entered anything into the Shields Down manually: including no "wildcards" that this commit seems to have addressed as a "fix."  Did wildcards somehow get automatically added, and break?  Ok, that makes sense - but how did that happen?

As a follow-up, I was able to resolve my issue with the instructions below - even though at first I had "nothing to clear."

It's the subtle but key dropdown menu on that Advanced tab, at the very top that SCROLLS AWAY when you SCROLL DOWN to look for and select "Sites and Shields Settings."  

You must SCROLL BACK UP that checkbox list, and change the `Time range` DropDownList to `All Time` to get all the sites to show up.  Then I had a list of 100+ sites that needed to be "cleared."

Most concerning though: Why do I have an exception list of over 100 sites?  That seems, unusually high when I barely ever disable a shield. And when I do disable shields, it's briefly until I flip it back on as it didn't fix the site.

This was not in any of the directions or suggested fixes.

**Comment by bridiver** [comment]
> Most concerning though: Why do I have an exception list of over 100 sites? That seems, unusually high when I barely ever disable a shield. And when I do disable shields, it's briefly until I flip it back on as it didn't fix the site.
> 

Toggling a shields value back to the original does not unset it, it just changes the value of the override. This is a long-standing issue I've wanted addressed and I believe @petemill is working on a change to differentiate between changing the override and unsetting it. 

As far as the actual cause, we don't know how this is happening so we added some debugging to give us more information to help track it down. Fortunately because there should never be a wildcard value for shields this was easy to address without knowing the root cause.

**Comment by MadhaviSeelam** [comment]
Brave | 1.41.95 Chromium: 103.0.5060.114 (Official Build) (64-bit)
-- | --
Revision | a1c2360c5b02a6d4d6ab33796ad8a268a6128226-refs/branch-heads/5060@{#1124}
OS | Windows 11 Version 21H2 (Build 22000.739)


Filed #23988

**Comment by LaurenWags** [comment]
Verification in progress with

```
Brave | 1.41.96 Chromium: 103.0.5060.114 (Official Build) (x86_64)
-- | --
Revision | a1c2360c5b02a6d4d6ab33796ad8a268a6128226-refs/branch-heads/5060@{#1124}
OS | macOS Version 12.4 (Build 21F79)
```

**Comment by leotm** [comment]
the issue returned for me on Windows on `Version 1.41.96 Chromium: 103.0.5060.114 (Official Build) (64-bit)`

my [suggestion](https://github.com/brave/brave-browser/issues/23214#issuecomment-1161711262) earlier no longer worked, i suspected a reinstall would, but thought must be an easier way

flicking it [off and on again](https://youtu.be/nn2FB1P_Mn8) did the trick again for me

![Capture](https://user-images.githubusercontent.com/1881059/179489870-f07f2c39-d331-4e27-8ef9-5672bd3a0551.PNG)

**Comment by Werve** [comment]
I confirm me too on 1.42.97 Chromium: 104.0.5112.102 (Build ufficiale) (a 64 bit) the shields are still default off on never seen websites.
